### DISCUSSION TITLE
Publish a GitHub Page with Jekyll Action, how to customize the page?

### DISCUSSION THREAD

**Discussion by kalasoo** [community/community#49275] [Actions] [answered]
### Select Topic Area

Question

### Body

I have setup a GitHub Page (https://jibenzhengchang.com/) with the Jekyll GitHub Action, which is very fluent. The repo link is: https://github.com/kalasoo/jibenzhengchang-home

After I published the site, I want to

1. Customize the theme
2. Remove the title added by default

How can I customize the Jekyll page (like CSS and layout) myself?

**Comment by airtower-luna** [comment, marked as answer]
Check the Jekyll documentation: https://jekyllrb.com/docs/

In short, you'll have to add config files, templates, etc. (depending on what you want) to tell Jekyll how you want your site to look.

**Reply by kalasoo** [reply]
As I just used the action to render the website pages, there's no branch like `gh-pages` or local Jekyll files. Where should I put the `_config.yml` and other templates files?

**Reply by airtower-luna** [reply]
Your workflow uses the root of your repository as the source of the Jekyll build, so that's where your Jekyll config needs to go, plus subdirectories where appropriate. :slightly_smiling_face:

**Reply by kalasoo** [reply]
Thanks, Let me try.

**Reply by kalasoo** [reply]
@airtower-luna it works! thank you very much!

**Reply by airtower-luna** [reply]
Glad I could help! Please don't forget to mark the answer. :slightly_smiling_face:
